### ISSUE TITLE
config parser chokes on a couple spanning tree lines

### ISSUE THREAD

**Issue by craigers521** [ecoen66/merakicat#6]
#### Behavior 
running `python merakicat.py check file <filename>` errors out when these 2 lines are in the cli output for a iosxe switch: 

```
spanning-tree portfast default
spanning-tree portfast bpduguard default 
```

#### Workaround
manual edit config file to remove those 2 lines then the parser works and merakicat finishes the check

#### environment
MacOS 14.4.1
Python 3.11.4

**Comment by ecoen66** [comment]
Opened [issue #11 for ciscoconfigparse2]( https://github.com/mpenning/ciscoconfparse2/issues/11) to resolve this.

**Comment by ecoen66** [comment]
@craigers521, Thanks for finding this!

Fixed in ciscoconfparse2 v0.7.50.
Fixed in merakicat v0.2.6.
